### ISSUE TITLE
Request - user management - sharing required reading

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8187]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

When granting a user more roles, it is a requirement to share the information behind the Scary Red Box:

<img width="392" alt="Screenshot 2024-10-10 at 08 35 27" src="https://github.com/user-attachments/assets/aa61c8e0-395b-45b2-a719-058b65aa251b">

but that still requires some explanation (shared roles should always be emphasized, etc.), and then copy/paste of several resources (see https://github.com/ArctosDB/arctos/issues/8184#issuecomment-2405451349 for example). Making all of this easier and more consistent would be good. How can we do that?

1. I could add a 'copy/paste this markdown' bit to https://arctos.database.museum/Admin/user_roles.cfm
2. We could recommend sending the link to (maybe a more public-friendly version of) https://arctos.database.museum/Admin/user_roles.cfm
3. Arctos could send notifications, which sounds
    * awesome, I think many problems involve supervisors ignoring all the requirements and leaving users in the dark, and
    * possibly a little overwhelming when one gets (and/or loses?) a bunch of rights/notifications, and
    * possibly cryptic (everyone DOES see and read notifications, right?!)
4. That superawesome thing I haven't thought of yet, help??

**Comment by dustymc** [comment]
Somehow integrate @mkoo 's document/map, https://docs.google.com/document/d/1MkD2neHjR99JXtTQ2KO17Hm1d-_ROdmpIyjSjNB8eps/edit?tab=t.0 (maybe a 'click to highlight {type} roles' widget)

Merge the various documentation into one column, add copy-link.

Add rows for 

* anonymous (user clicked the "I'll be nice" button)
* public (user has created a non-operator account)

These things are different, that should be documented somewhere, with all other access-stuff seems right

Try to more-hide the admin stuff - an increasing number of roles will only ever be carried by a very few people, try to make them a bit less intrusive (but they still need to be there, someone who wants to control who gets bug reports should be able to figure it out).

Going dev.
